src/components/Home.jsx

```diff
@@ -80,11 +86,15 @@ class Home extends React.Component {
           <img role="presentation" className="divider" src={Divider} />
           <div className="home-page__body-text">
             <b className="body-copy-first-word">Welcome.</b>{' '}
-            {this.props.project.description}
+            {this.props.project && this.props.project.description}
```

Comment:
Existential checks were also added in a couple places in case the project has not yet been retrieved.

Reply:
👍